torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1875,7 +1860,14 @@ ModuleMethodVector InsertQuantDeQuantHelper::getInvokedMethods(
         if (module_instance == graph->inputs()[0]) {
           m = module;
         } else {
-          m = findChildModuleToQuantize(module, module_instance);
+          TORCH_INTERNAL_ASSERT(
+              module_instance->node()->kind() == prim::GetAttr,
+              "Module instance should come from GetAttr.");
+          if (module_method_name.find("_observer_") == std::string::npos) {
```

Comment:
I think you need to check the child instance name here, see the original function

Reply:
isn't it the same? child_instance is the same as module_instance. And we already get the name of the node here.

Reply:
module_method_name is the name of the method here, e.g. "forward", I think you'll need `module_instance->node()->s(attr::name)`